### DISCUSSION TITLE
Different transition between paths

### DISCUSSION THREAD

**Discussion by andresclua** [swup/swup#723] [Help] [answered]
How can I implement distinct transitions for the paths /work/ and /work/{slug}? The current configuration doesn't seem to be working as expected. Here's the code:
```js
{
  from: '(.*)',
  to: '/work',
  in: async (next, infos) => {

  },
  out: (next, infos) => {

  },
},
{
  from: '(.*)',
  to: '/work/(.*)',
  in: async (next, infos) => {

  },
  out: (next, infos) => {

  },
}

```
Thanks for the help, pretty sure is super simple

**Comment by hirasso** [comment, marked as answer]
This is a [JS Plugin](https://github.com/swup/js-plugin) question.

The plugin's `from` and `to` paths are being parsed by [path-to-regexp](https://github.com/pillarjs/path-to-regexp).  Does the following work for you?

```js
{
  from: '(.*)',
  to: '/work/',
  in: (next, infos) => {

  },
  out: (next, infos) => {

  },
},
{
  from: '(.*)',
  to: '/work/:slug',
  in: (next, infos) => {

  },
  out: (next, infos) => {

  },
}
```

**Reply by andresclua** [reply]
TSM!
